core/benches/bench_bitstash.rs

```diff
@@ -92,6 +121,14 @@ fn bench_empty_cache(c: &mut Criterion) {
         let mut group = c.benchmark_group("Bench: `fill_bitstash` (empty cache)");
         group
             .bench_function("fill_bitstash", |b| b.iter(|| empty_cache::fill_bitstash()));
+
+        group.bench_function("one_put", |b| {
+            b.iter_batched(
+                || create_large_stash(),
+                |mut stash| empty_cache::one_put(&mut stash),
```

Comment:
Not testing empty cache.

Reply:
You need to pull a new instance from the storage for every iteration to make it have empty cache. Can't re-use instances otherwise we populate cache.